pkg/nsx/services/subnet/subnet_test.go

```diff
@@ -398,7 +398,7 @@ func TestSubnetService_UpdateSubnetSet(t *testing.T) {
 	})
 
 	patchesCreateOrUpdateSubnet := gomonkey.ApplyFunc((*SubnetService).createOrUpdateSubnet,
-		func(r *SubnetService, obj client.Object, nsxSubnet *model.VpcSubnet, vpcInfo *common.VPCResourceInfo) (*model.VpcSubnet, error) {
+		func(r *SubnetService, obj client.Object, nsxSubnet *model.VpcSubnet, vpcInfo *common.VPCResourceInfo, isUpdate bool) (*model.VpcSubnet, error) {
```

Comment:
Do you think we need a new UT for `createOrUpdateSubnet`?

Reply:
Yes, some part of the code in createOrUpdateSubnet is not covered yet. Added the unit test, thanks